src/test/java/com/radixdlt/engine/RadixEngineTest.java

```diff
@@ -54,14 +54,15 @@ public void when_validating_an_atom_with_particle_which_conflicts_with_virtual_s
 		when(constraintMachine.validate(any())).thenReturn(Optional.empty());
 		EngineStore engineStore = mock(EngineStore.class);
```

Comment:
EngineStore missing a type argument, which causes Eclipse compiler to type-fail on RadixEngine construction below.